From f9634b82a18e4041971cdf8be2917eb5a27962b8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Sun, 23 May 2021 13:36:29 +0200
Subject: [PATCH] AssetManager2: Align creation of FindEntryResult with
 declaration

GCC 10.X complains about this and refuses to compile otherwise.
---
 libs/androidfw/AssetManager2.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/androidfw/AssetManager2.cpp b/libs/androidfw/AssetManager2.cpp
index bec80a7d605e..250032e12e4d 100644
--- a/libs/androidfw/AssetManager2.cpp
+++ b/libs/androidfw/AssetManager2.cpp
@@ -821,11 +821,11 @@ base::expected<FindEntryResult, NullOrIOError> AssetManager2::FindEntryInternal(
     .entry = *entry,
     .config = *best_config,
     .type_flags = type_flags,
+    .dynamic_ref_table = package_group.dynamic_ref_table.get(),
     .package_name = &best_package->GetPackageName(),
     .type_string_ref = StringPoolRef(best_package->GetTypeStringPool(), best_type->id - 1),
     .entry_string_ref = StringPoolRef(best_package->GetKeyStringPool(),
                                       best_entry->key.index),
-    .dynamic_ref_table = package_group.dynamic_ref_table.get(),
   };
 }
 
